larq_compute_engine/mlir/BUILD

```diff
@@ -293,14 +293,20 @@ pybind_extension(
 
 py_library(
     name = "converter",
-    srcs = ["python/converter.py"],
+    srcs = [
+        "python/converter.py",
+        "python/schema_py_generated.py",
```

Comment:
I added an autogenerated tflite flatbuffer schema file since it is not included in the pip package of earlier versions of TensorFlow and is also missing on the macos builds of TensorFlow.

I tried using https://github.com/jackwish/tflite as well, but it doesn't include the object-based flatbuffer API for mutations and adding the API to the package would conflict with their import structure.